src/components/questions.tsx

```diff
@@ -182,54 +191,79 @@ function ChoosePosition(props: {
-                `${
-                  active
-                    ? 'ring-2 ring-white ring-opacity-60 ring-offset-2 ring-offset-sky-300'
-                    : ''
-                }
-              ${checked ? 'bg-sky-900 bg-opacity-75 text-white' : 'bg-white'}
-                relative flex cursor-pointer rounded-lg px-5 py-4 shadow-md focus:outline-none`
+              value={pos}
+              className={({ checked }) =>
+                clsx(
+                  checked ? 'ring-purple-300' : 'ring-gray-200',
+                  'ring-2 relative flex cursor-pointer rounded-lg px-5 py-4 focus:outline-none'
+                )
               }
             >
               {({ checked }) => (
-                <span className={checked ? 'bg-blue-200' : ''}>
-                  {index === 0 ? 'YES: ' : ''}
-                  {bigintStringify(pos)}
-                </span>
+                <div className="flex h-46 w-20 items-center justify-center gap-x-2">
+                  <motion.div layout="position" className="text-sm">
+                    <RadioGroup.Label
+                      as="p"
+                      className="font-medium text-gray-900"
+                    >
+                      {index === 0 ? 'YES' : 'NO'}
+                    </RadioGroup.Label>
+                  </motion.div>
+                  {checked && (
+                    <motion.div
+                      className="shrink-0 text-gray-900"
+                      initial={{ scale: 0 }}
+                      animate={{ scale: 1 }}
+                      transition={{ type: 'spring' }}
+                    >
+                      <FiCheck className="h-5 w-6" />
+                    </motion.div>
+                  )}
+                </div>
               )}
             </RadioGroup.Option>
           ))}
         </div>
       </RadioGroup>
-      <input
-        type="submit"
-        value="Submit vote"
-        disabled={!position}
-        className="btn-primary p-1 rounded mt-2"
-      />
+      <div className="mt-2 px-2 flex justify-end">
+        <input
+          type="submit"
+          value="Submit vote"
+          disabled={!position}
+          className={clsx(
+            'btn-primary rounded mt-2 p-2',
+            position ? 'cursor-pointer' : 'cursor-not-allowed'
+          )}
+        />
+      </div>
     </form>
   );
 }
 
-export function VoteOnLatestQuestion(props: { ecOfferId: number }) {
+export function VoteOnLatestQuestion(props: {
+  ecOfferId: number;
+  instance?: [property: string, value: RpcRemote][];
+}) {
   const walletUtils = useContext(WalletContext);
   const { status, data } = usePublishedDatum(
     'committees.Economic_Committee.latestQuestion'
   );
 
   console.debug('render VoteOnLatestQuestion', status, data);
   if (!data?.positions) {
-    return <b>{status} for a question</b>;
+    return (
+      <b>
+        {status.charAt(0).toUpperCase() + status.slice(1)} for a question...
```

Comment:
longing for lodash https://lodash.com/docs/4.17.15#capitalize

probably not worth taking the dep

Reply:
Yea just pulled this into a helper. I'd consider lodash for more logic-heavy helpers but not this